### DISCUSSION TITLE
Milestones are moved

### DISCUSSION THREAD

**Discussion by frankulatowski-scm** [Leantime/leantime#625] [Q&A] [answered]
I am a member of several projects. Today I opened project B. 

In the meantime, I receive an email notification that a milestone in project A has been changed. I click on the link, the dialog box "Milestone" appears and I change something in this box. 

Now something unexpected happens: When I save, this milestone is moved from its original project A to project B, where I am currently in.

Is this intended?

Regards, Frank

**Comment by marcelfolaron** [comment]
Thanks.
No this is not expected. Clicking the link should change the project so
that this does not happen.

Can you file a bug for that?

**Reply by frankulatowski-scm** [reply]
Yes, I will do that.

**Comment by marcelfolaron** [comment, marked as answer]
Thank you!
